cmd/nginx-ingress/main.go

```diff
@@ -47,11 +48,13 @@ var (
 )
 
 const (
-	nginxVersionLabel      = "app.nginx.org/version"
-	versionLabel           = "app.kubernetes.io/version"
-	appProtectVersionLabel = "appprotect.f5.com/version"
-	agentVersionLabel      = "app.nginx.org/agent-version"
-	appProtectVersionPath  = "/opt/app_protect/RELEASE"
+	nginxVersionLabel        = "app.nginx.org/version"
+	versionLabel             = "app.kubernetes.io/version"
+	appProtectVersionLabel   = "appprotect.f5.com/version"
+	agentVersionLabel        = "app.nginx.org/agent-version"
+	appProtectVersionPath    = "/opt/app_protect/RELEASE"
+	appProtectBundleFolder   = "/etc/nginx/waf/bundles/"
+	appProtectv5BundleFolder = "/etc/app_protect/bundles/"
```

Comment:
@oseoin @jjngx does the path for logconf bundle changes based on versions or its same as

Reply:
@vepatel `appProtectv5BundleFolder` is used for v5, otherwise `appProtectBundleFolder` is used

Reply:
Just updated naming to make the switch obvious